### ISSUE TITLE
TV Shows not seen kodi.

### ISSUE THREAD

**Issue by BTopbas** [mancuniancol/script.subscription.quasar.trakt#2]
I love trakt subscription and used long time with pulsar. Now i delete all library and pulsar. installed quasar and create new library tv shows and movies. And i use this plugin. Movies added library with no problem. But tv shows cant. Kodi not seen tv shows. I looked folder tv shows names and strm files is okey. eveything is look okey. But every library refresh and scan for new videos kodi not seem this. Any idea why? I love subscription plugin. Because The advantage is that you have all the tools from local library as trailer, cinema experience, etc. that you say..

**Comment by mancuniancol** [comment]
I know what you mean, but trakt is natively supported by Quasar. This script is deprecated :(

**Comment by mancuniancol** [comment]
https://github.com/scakemyer/plugin.video.quasar/issues/31

**Comment by BTopbas** [comment]
I got it. But quasar dont create local library right? I cant make it. I select library folder on quasar settings. And i create library with this folder. Nothing happening folder is empty. I also walk a round quasor menus. But not luck. How can add strm files to library? Most people likes and uses your subscription script. This was perfect. :(

**Comment by mancuniancol** [comment]
Yes, it does and the use tratk API. I will check the script, though

**Comment by ShlomiD83** [comment]
I checked it out yesterday, and quasar trakt list subscription worked great.

**Comment by ShlomiD83** [comment]
Use TMDB scraper for tv shows, TVDB isn't working with trakt for some reason.

**Comment by mancuniancol** [comment]
I've found that the script was time-out.  I've done some changes in the timing and it is working again in my side.  To get the new changes, you have to download the mctools module https://github.com/mancuniancol/script.module.mctools/archive/v1.3.7.zip

Please, let me know if it works or not.
